autoload/venu/native.vim

```diff
@@ -93,32 +102,32 @@ function! venu#native#list()
         let l:prevdepth = l:depth
 
       else
-        " let l:indent = l:line[0]
-        let l:mode = l:line[1]
-        let l:cmd = l:line[2]
-        let l:stack[-1].mode[l:mode] = l:cmd
+        " Ignore indent (l:line[0]) since it's not used
+        let l:mode      = l:line[1]
+        let l:mode_flag = l:line[2]
+        let l:cmd       = l:line[3]
+        let l:stack[-1].mode[l:mode] = {'flag': l:mode_flag, 'cmd': l:cmd}
       endif
     endif
   endfor
   return l:root
 endfunction
 
-function! s:debugTraverse(l, d)
-  let l:s = ''
+function! s:DebugTraverse(entry, depth) abort
```

Comment:
What is the `depth` supposed to do? As far as i can see it simply increases the indentation in front of the printed output (of all rows). Intuitively it sounds like it would control how deep the submenus are printed (e.g. `depth=1` would only print until the first submenu of each top level menu, but not traverse further down the into submenus of submenus).

Reply:
It's more like `indent` actually. It just keeps how much space to be printed before each line. I will rename it to `indent`.